Unified.py

```diff
@@ -0,0 +1,120 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""
+Created on Sun Nov 14 12:30:49 2021
+
+@author: DianaG
+"""
+
+import ldap
+import csv
+import pandas as pd
+import datetime
+import dateutil.relativedelta
+
+# **** Part 1/2 ****
+
+ldap_server = 'ldap://172.20.1.216:389'
+base_dn = 'DC=lab, DC=visiblerisk, DC=com'
+ldap_login = 'Administrator'
+ldap_pass = 'Nhwk2iISxJuOZQa43IDch0J8z0hqB5jY'
+search_object = '(&(objectClass=user)(objectCategory=person))'
+search_attr = ['userAccountControl', 'lastLogon', 'PwdLastSet', 'memberOf']
+
+
+conn = ldap.initialize(ldap_server)
+conn.set_option(ldap.OPT_REFERRALS, 0)           
```

Comment:
What is 0 here? consider replacing it with a const, or alternatively add a comment about what this line does